Drivers/SensorFusion/Src/MahonyAHRS.cpp

```diff
@@ -63,15 +63,15 @@ void MahonyAHRSupdate(float gx, float gy, float gz, float ax, float ay, float az
 
 		// Normalise accelerometer measurement
 		recipNorm = invSqrt(ax * ax + ay * ay + az * az);
-		ax *= recipNorm;
-		ay *= recipNorm;
-		az *= recipNorm;     
+		ax = ax * recipNorm;
+		ay = ay * recipNorm;
+		az = az * recipNorm;     
```

Comment:
I'm curious if you know what exactly the impact is that this has on the system and why the previous operation was giving a build warning? I didn't look to far into it.

Reply:
seems like += works differently from = then +. it is unsafe to actually use += for volatile values for some reason